spec/features/allocate_feature_spec.rb

```diff
@@ -19,55 +20,64 @@
 
   let!(:case_information) {
     create(:case_information, offender: build(:offender, nomis_offender_id: nomis_offender_id), tier: 'A', case_allocation: 'NPS', probation_service: 'England')
-    create(:case_information, offender: build(:offender, nomis_offender_id: never_allocated_offender))
+    create(:case_information, offender: build(:offender, nomis_offender_id: never_allocated_offender_id))
   }
 
   before do
     signin_spo_user
   end
 
-  scenario 'accepting the recommended POM type', vcr: { cassette_name: 'prison_api/create_new_allocation_feature' } do
-    visit prison_prisoner_staff_index_path('LEI', nomis_offender_id)
+  context 'when a journey begins on the "Make new allocations" page' do
+    before do
+      visit unallocated_prison_prisoners_path('LEI')
+
+      # Takes you to the Review case and allocate page where users review case details and scroll down to select from a list of POMs
+      click_link unallocated_offender_name
+      expect(page).to have_current_path(prison_prisoner_staff_index_path('LEI', never_allocated_offender_id))
```

Comment:
...but it looks like this change is probably the culprit!

The previous test loaded the page:

```ruby
prison_prisoner_staff_index_path('LEI', nomis_offender_id)
```

whereas now it loads the page:

```ruby
prison_prisoner_staff_index_path('LEI', never_allocated_offender_id)
```

We're actually allocating to a different offender. Previously we allocated the offender with ID `nomis_offender_id`, whereas now we're using `never_allocated_offender_id`. I'm guessing that they've got different tiers, so need a different type of POM.

---

This would explain why the "accepting the recommended POM type" journey ended up looking like it was actually taking the "override" journey. Because we were allocating a different offender than before, so getting a different POM type recommendation.

Reply:
@ollietreend Oh goodness! Thank you Ollie! I can't believe I missed that. Although I am sure I originally copied that url and then changed it because it didn't work. It is too long ago to remember!